src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SubjectAccessRequestServiceTest.kt

```diff
@@ -42,6 +43,9 @@ class SubjectAccessRequestServiceTest {
   @Mock
   private lateinit var actionPlanService: ActionPlanService
 
+  @Mock
```

Comment:
There has been a lot of refactoring of this test class, not only to make it work now that the impl uses the NoteService to get the goal notes; but also to refactor the tests themselves to make them more readable